### PR TITLE
Mentor page

### PR DESCRIPTION
**Pull request by hlaineka**
Mentor page is divided into few components, mentor search, mentor skills and mentor cards need to be added as one. css for search and skills is added to this version. Still some problems with lint and jest configurations.

### CODE REVIEW

File: src/features/MentorPage/MentorPage.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	flex-wrap: wrap;
+	padding: 2rem 0;
+	width: 90%;
+	max-width: 90%;
+	margin: auto;
+`;
+
+export const StyledMentorCardContainer = styled.div`
+	flex: 1;
+`;
```
Comment (rottabonus) [inline src/features/MentorPage/MentorPage.styles.tsx:63]: Not a fan of one file that has dozens of components.
As a developer it is easier to check the file-strucure and then see what kind of building blocks the application consists of. Then its nice to see a short definition of the component style. If there is a lot of these - it becomes just a .css file, and then why not use css in the first place

Comment (rottabonus) [reply src/features/MentorPage/MentorPage.styles.tsx:63]: Sidenote: I dont think putting _Styled_, in front of the component is needed.

Comment (hlaineka) [reply src/features/MentorPage/MentorPage.styles.tsx:63]: I'm under the impression that these would be moved to storybook somehow at some point. How would you do this now that we are waiting for that to be possible?

Comment (rottabonus) [reply src/features/MentorPage/MentorPage.styles.tsx:63]: I guess we dont need to put every individual styled div into the Storybook. More like some components that have interactivity in them such as Buttons, Lists, etc.

File: src/components/Chip/Chip.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { ChipProps } from './types';
+import styled from 'styled-components';
+
+/**
+ * Layout to one Skill Chip
+ */
+
+// eslint-disable-next-line react/prop-types
```
Comment (aeniinik) [inline src/components/Chip/Chip.tsx:8]: This probably needs to be fixed in linting rules.

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Ylitse MentorApp - SOS-Lapsikylä</title>
-    <link rel="stylesheet" type="text/css" href="/static/styles/fonts.css"/>
-    <link rel="stylesheet" type="text/css" href="/static/styles/styles.css"/>
+    <title>React Typescript Webpack</title>
```
Comment (aeniinik) [inline src/index.html:7]: The title seems to change (see master).

**Comment by hlaineka** [comment]
Openin a non-draft PR

**Pull request closed without merging**